kkahrs/magma
# [lte][agw] allow pydep to build packages for multiple base OSes

## Summary
* add an `override` section to magma.lockfile to track dependencies of non-default OS releases
* workaround broken upstream package versioning for aioeventlet on ubuntu focal
* add `vm` keyword to package() in `fabfile.py`
* fix python interpreter in `build-magma.sh` for python 3.8
* remove unneeded `pyvenv` role


## Test Plan
* verify `fab dev package:vcs=git` generates packages on default debian stretch vm
* verify `fab dev package:vcs=git,vm=magma_focal` generates packages on new ubuntu focal vm

## Additional Information
